lib/time/duration.ex

```diff
@@ -424,12 +545,25 @@ defmodule Timex.Duration do
   end
 
   @doc """
-  Time interval since Epoch.
+  Returns the amount of time since Epoch.
+
+  The argument is an atom indicating the type of time units to return.
+
+  The allowed unit type atoms are:
+  - :microseconds
+  - :milliseconds
+  - :seconds
+  - :minutes
+  - :hours
+  - :days
+  - :weeks
+
+  ## Examples
 
-  The argument is an atom indicating the type of time units to return (see
-  convert/2 for supported values).
+  iex> Timex.Duration.now(:seconds)
+  :os.system_time(:seconds)
 
-  When the argument is omitted, the return value's format is { megasecs, seconds, microsecs }.
+  When the argument is omitted or nil, a Duration is returned.
```

Comment:
I would love to have a doctest after this, but I don't know of a way to write an elegant one.
```
iex> Timex.Duration.now()
:os_system_time(:micro_seconds) |> from_microseconds
```

would be purely tautological as well, and does a worse job than the above tautological test at explaining how this works. I don't like it. Any thoughts on a better way to do this?